source/index.rst

```diff
@@ -31,75 +31,82 @@
    basic-website
    advanced-features
 
-Redact is a set of applications that allows developers to build end-to-end
-encrypted websites and users to use these websites. When leveraging Redact,
-website owners are able to display secure user data on their webpages without
-being able to know or read that data back to themselves. Securing websites in
-this way is beneficial to both the user and the website owner as it protects
-from data leaks, allows complete control and knowledge over who has acces to
-data, and standardizes secure authentication mechanisms that don't rely on
-passwords.
-
-.. note:: The two applications that enable this functionality are
+Redact is a set of applications that allows developers to build websites with
+end-to-end encrypted data. When leveraging Redact, website owners are able to
+display secure user data on their webpages without having access to that data
+themselves. Instead, users store sensitive data in their personal Redact storage
+and retain the ability to grant and revoke access to other users of
```

Comment:
Don't like how this is re-written because it breaks up what was supposed to be the thesis statement. The thesis being, using Redact is beneficial, it's beneficial because A, B, C, and then my supporting paragraphs are explaining why A, B, and C are beneficial.